src/frontend/src/flows/faq/index.ts

```diff
@@ -38,39 +56,45 @@ const pageContent = html`
         background-color: transparent;
       }
       50% {
-        background-color: var(--grey-100);
+        background-color: var(--rainbow-orange);
+        border-radius: 0.3em;
       }
       100% {
         background-color: transparent;
       }
     }
-
-    a {
-      color: var(--grey-800);
-    }
-
-    ul {
-      list-style-type: none;
-    }
-
-    li {
-      border-radius: 0.5rem;
-      padding: 0.5rem;
-    }
     :target {
       animation-name: flash-question;
-      animation-duration: 3s;
+      animation-duration: 600ms;
     }
   </style>
-  <div class="container" id="faq">
-    <h1>FAQ</h1>
-    <ul>
-      ${Object.values(questions).map((faq) => renderQuestion(faq))}
+  <div class="faq__container">
+    <h1 class="faq__title">FAQ</h1>
+    <ul class="faq__questions">
+      ${questionsArray.map((faq) => renderQuestion(faq))}
     </ul>
   </div>
 `;
 
+// Open the anchor with id="foo" if the page hash is "#foo"
+const openAnchor = (): void => {
+  const hash = location.hash.substring(1);
+
+  if (hash !== "") {
+    const details = document.getElementById(hash);
+    console.log(details);
+
+    if (details) {
+      details.setAttribute("open", "");
+    }
+  }
+};
+
 export const faqView = (): void => {
+  document.title = "FAQ | Internet Identity";
   const container = document.getElementById("pageContent") as HTMLElement;
   render(pageContent, container);
+  openAnchor(); // needs to happen after DOM was rendered
 };
+
+faqView();
```

Comment:
Shouldn't this be called from `index.ts`? I'm not sure what it means to have the call at the top-level on the module

Reply:
@nmattia Once you've removed this call, I'm happy to approve.

Reply:
yup, just double checking what this is about!

Reply:
I know. I used a custom index at some point. Removing now.